src/containers/PipelineRun/PipelineRun.js

```diff
@@ -35,7 +35,7 @@ import { fetchClusterTasks, fetchTasks } from '../../actions/tasks';
 import { fetchTaskRuns } from '../../actions/taskRuns';
 import { rerunPipelineRun } from '../../api';
 
-import '../../components/Run/Run.scss';
+import '../../../packages/components/src/scss/Run.scss';
```

Comment:
should be able to reference this as `'@tektoncd/dashboard-components/dist/scss/Run.scss'`
not ideal to rely on the dist folder but we already have it in a couple of places so at least it's consistent
we can review them all together once we decide on a long term solution (likely having a layout component that includes a set of base styles common to multiple pages)